### PR TITLE
Std2isis gtests

### PR DESCRIPTION
**Pull request by Kelvinrr**
## Description

adds std2isis gtests

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [ ] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [ ] I have added any user impacting changes to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [ ] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

**Comment by Kelvinrr** [comment]
Still on the fence on some of these tests, Argb test was made using a png image as nothing about that param seems to require a jpeg (if anything, using a jpeg makes little sense as iirc jpegs can't have an alpha channel). But, should I add a jpeg test as a way to simply track the error? 

If so, I will add them as unit tests to the importer and not append this PR.

**Comment by kaitlyndlee** [comment]
i think adding the error checking makes sense. Everything looks good though.

**Review by kaitlyndlee — DISMISSED** [review]

File: isis/tests/ImageImporterTests.cpp [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+#include "gmock/gmock.h"
+
+using namespace Isis;
+
+// Expected fail on MacOS
+TEST_F(TempTestingFiles, DISABLED_UnitTestImageImporterTestJpeg) {
+  FileName inputName("data/stdFormatImages/rgb.jpg");
+
+  ImageImporter *importer = ImageImporter::fromFileName(inputName);
```
Comment (Kelvinrr) [inline isis/tests/ImageImporterTests.cpp:19]: Jpeg test added as disabled

File: isis/src/base/apps/std2isis/std2isis.h [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#ifndef std2isis_h 
+#define std2isis_h
+
+#include "Cube.h"
```
Comment (AustinSanders) [inline isis/src/base/apps/std2isis/std2isis.h]: We should be able to get rid of the cube import here

File: isis/tests/FunctionalTestsStd2Isis.cpp [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+#include "std2isis.h"
+
+using namespace Isis;
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/std2isis.xml").expanded();
+
+TEST_F(TempTestingFiles, FunctionalTestsStd2IsisDefault) {
+  QString outCubeFileName = tempDir.path() + "/outTemp.cub";
+  
+  QVector<QString> args = {"from=data/stdFormatImages/rgb.png", "to="+outCubeFileName, };
```
Comment (AustinSanders) [inline isis/tests/FunctionalTestsStd2Isis.cpp]: Will this trailing comma cause any issues?

Comment (Kelvinrr) [reply isis/tests/FunctionalTestsStd2Isis.cpp]: don't think so, at least it has caused no issues but I might as well remove it.

**Review by tgiroux — APPROVED** [review]
Looks good to me

**Pull request merged**